go/enclave/rollupchain/rollup_chain.go

```diff
@@ -159,11 +167,12 @@ func (rc *RollupChain) newBlockSubmissionResponse(bs *obscurocore.BlockState, ro
 		head = headRollup.Header
 	}
 	return common.BlockSubmissionResponse{
-		BlockHeader:    headBlock.Header(),
-		ProducedRollup: rollup,
-		IngestedBlock:  true,
-		FoundNewHead:   bs.FoundNewRollup,
-		RollupHead:     head,
+		BlockHeader:        headBlock.Header(),
+		ProducedRollup:     rollup,
+		IngestedBlock:      true,
+		FoundNewHead:       bs.FoundNewRollup,
+		RollupHead:         head,
+		SubscribedReceipts: receipts,
```

Comment:
@tudor-malene Here and elsewhere, shouldn't this specifically be logs, and not the entire receipt?

Reply:
yes, should be logs everywhere.
changing now.